packages/webdriver/tests/request.test.js

```diff
@@ -72,6 +72,17 @@ describe('webdriver request', () => {
             expect(req.emit).toBeCalledWith('response', { result: expectedResponse })
         })
 
+        it('should short circuit if request throws a stale element exception', () => {
+            const req = new WebDriverRequest('', '')
+            req.emit = jest.fn()
+
+            const opts = Object.assign(req.defaultOptions, { uri: { path: '/wd/hub/failing' } })
+            expect(req._request(opts, 2)).rejects.toEqual(new Error('Error: Could not send request'))
+
+            const
+
```

Comment:
All three lines above can be removed

Reply:
Sorry - this was still a WIP... i got called into a meeting before i was able to put a comment on it.

Reply:
I think this test is done.